phylotoast/util.py

```diff
@@ -63,37 +60,35 @@ def parseFASTA(fastaFNH):
         if line[0] == ">":
             # conclude previous record
             if seq:
-                recs.append(FASTARecord(ID, descr, "".join(seq)))
+                recs.append(FASTARecord(seqID, descr, "".join(seq)))
                 seq = []
             # start new record
-            line = line[1:].split()
-            ID, descr = line[0], " ".join(line[1:])
+            line = line[1:].split(None, 1)
```

Comment:
None is the default for the initial sep parameter. Should edit this to .split(maxsplit=1) for clarity.